frontend/src/components/elements/CodeBlock/CodeBlock.scss

```diff
@@ -21,6 +21,15 @@
   See https://prismjs.com/download.html#themes=prism&languages=markup+css+clike+javascript
 */
 
+div.stCodeBlock {
+  margin: 0 0 1em 0;
+  position: relative;
```

Comment:
Explain with a comment why these are needed: "code blocks can be used used inside an Alert ..."